scripts/itkdev-docker-compose

```diff
@@ -16,6 +16,88 @@ function find_script_dir {
 script_dir=$(find_script_dir)
 script_path=$script_dir/$(basename "${BASH_SOURCE[0]}")
 
+function template_install {
+  local force=0
+  local list=0
+  local name=""
+
+  for var in "$@"
+  do
+    case "$var" in
+      --force)
+        force=1
+        ;;
+      --list)
+        list=1
+        ;;
+      *)
+        name="$var"
+        ;;
+    esac
+  done
+
+  local templates_dir=$script_dir/../templates
+  local source_dir=$templates_dir/$name/
+  local target_dir=$PWD
+
+  if [ "$list" -eq 1 ]; then
+    echo Templates:
+    for name in $templates_dir/*; do
+      echo "  $(basename $name)"
+    done
+    exit 0
+  fi
+
+  if [ -z "$name" ]; then
+    (>&2 echo "${bold}Missing template name${normal}")
+    exit 1
+  fi
+  if [ ! -d $source_dir ]; then
+    (>&2 echo "${bold}Invalid template name: $name (use --list to show valid template names)${normal}")
+    exit 1
+  fi
+
+  if [ "$force" -ne 1 ]; then
+    if [ -e $target_dir/docker-compose.yml ]; then
+      (>&2 echo "${bold}Target $target_dir/docker-compose.yml already exists (use --force to overwrite)${normal}")
+      exit 1
+    fi
+    if [ -e $target_dir/.docker ]; then
+      (>&2 echo "${bold}Target $target_dir/.docker already exists (use --force to overwrite)${normal}")
+      exit 1
+    fi
+  fi
+
+  echo "${bold}Installing $name template${normal}"
+  rsync --archive $source_dir $target_dir
+
+  local project_name_default=$(basename $target_dir)
+
+  # Help user set up a .env file
+  if [ ! -e $target_dir/.env ]; then
+    local project_name
+    local domain
+    read -p "Project name ($project_name_default)? " -e project_name
+    if [ -z "$project_name" ]; then
+      project_name=$project_name_default
+    fi
+    read -p "Domain? " -e domain
+    cat > $target_dir/.env <<EOF
+COMPOSE_PROJECT_NAME=$project_name
+COMPOSE_DOMAIN=$domain
```

Comment:
Should we suggestion a domain name ending in `local.itkdev.dk` to try to have the same setup for docker. And/or should domain also be optional?

Reply:
I was thinking about this as well, but ended up letting the user decide. Maybe it's better to decide (suggest something) for him.

Reply:
Fixed.